### PR TITLE
Add back the activity tab for projects

### PR DESCRIPTION
**Pull request by chiaberry**
https://github.com/cityofaustin/atd-data-tech/issues/16836

The activity tab should be on the projects page, but not the product page:

A projects page: 
https://deploy-preview-90--atd-product.netlify.app/projects/4937?tab=activity

A product page:
https://deploy-preview-90--atd-product.netlify.app/products/5086

### CODE REVIEW

File: components/shared/indexIssueDetails/Comments.js
Comment (chiaberry) [inline components/shared/indexIssueDetails/Comments.js:1]: just bringing this deleted file back

**Comment by amenity** [comment]
Thanks for the speedy work, @chiaberry -- cc/ @TracyLinder

**Review by amenity — APPROVED** [review]
💃🏻 🪄 🎩  📰 ... 🙏🏻 🚢

**Review by roseeichelmann — APPROVED** [review]
:magic_wand:

**Pull request merged**